11/main.js

```diff
@@ -0,0 +1,294 @@
+    productPrice: 254,
+  },
+  {
+    productName: "Creatine",
+    productData: {
+      weight: 123,
+      certificate: false,
+      dateOfExpiry: "04/12/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice:134,
+  },
+  {
+    productName: "BCA",
+    productData: {
+      weight: 121,
+      certificate: true,
+      dateOfExpiry: "03/01/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "Francia",
+    productPrice: 16,
+  },
+  {
+    productName: "Milk",
+    productData: {
+      weight: 96,
+      certificate: false,
+      dateOfExpiry: "16/06/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 16,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: false,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 145,
+  },
+  {
+    productName: "Jagermeister",
+    productData: {
+      weight: 56,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "UK",
+    productPrice: 115,
+  },
+  {
+    productName: "Spaghetti",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "Italia",
+    productPrice: 16,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  {
+    productName: "Protein",
+    productData: {
+      weight: 96,
+      certificate: true,
+      dateOfExpiry: "21/10/22",
+      sugarFree: true,
+    },
+    productProvider: "FOP Skyba R.R.",
+    productCountry: "USA",
+    productPrice: 211,
+  },
+  
+];
+let i = 0;
+const div = document.createElement("div");
```

Comment:
It's better to give more meaningful names to variables, for example
```suggestion
const root = document.createElement("div");
```